modules/snooker/src/test/ts/browser/HeaderOperationsTest.ts

```diff
@@ -3,278 +3,828 @@ import * as TableOperations from 'ephox/snooker/api/TableOperations';
 import * as Assertions from 'ephox/snooker/test/Assertions';
 
 UnitTest.test('HeaderOperationsTest', function () {
```

Comment:
This file is _long_. I'm undecided on whether it's better as one file or if we should break it into multiple. I guess, if anyone else chimes in they can tie-break, I don't mind if it's left as is for now.